cluster/src/main/scala/org/apache/spark/memory/SnappyUnifiedMemoryManager.scala

```diff
@@ -635,6 +676,23 @@ object SnappyUnifiedMemoryManager extends Logging {
   }
 
   /**
+    * Return a fraction which determines what is the max limit storage can grow.
+    */
+  def getStorageEvictionPercent(conf: SparkConf): Float = {
+    val cache = Misc.getGemFireCacheNoThrow
+    if (cache ne null) {
+      val thresholds = cache.getResourceManager.getHeapMonitor.getThresholds
+      if (thresholds.getEvictionThreshold > 0.1f) {
+        cache.getResourceManager.getHeapMonitor.getThresholds.getEvictionThreshold
+      } else {
+        DEFAULT_EVICTION_PERCENT
+      }
+    } else {
+      conf.getDouble("spark.testing.maxStorageFraction", DEFAULT_EVICTION_PERCENT).toFloat
```

Comment:
Property is a fraction which is getting mixed with percent. Use one consistent name to avoid confusion in this class -- perhaps fraction throughout since that is what Spark uses.